### DISCUSSION TITLE
sample.yaml question about assembly

### DISCUSSION THREAD

**Discussion by akashjinandra** [usnistgov/ARIAC#209] [Q&A] [answered]
Hello again, when looking at the sample.yaml inside the trials for ariac gazebo package, it specifies an assembly order: 
![image](https://user-images.githubusercontent.com/22859817/224578596-a595403b-3870-404f-ba9d-44d7940a8af2.png)

However based on this: 
![image](https://user-images.githubusercontent.com/22859817/224578644-b0602fd2-8df8-4a7a-8e9a-cdd1ced21a67.png)

It would seem that we don't already have pre-spawned parts for this assembly task. Is there something I'm missing here or would this challenge not be possible to complete?

**Comment by jaybrecht** [comment, marked as answer]
Yes for an actual assembly order the parts should be present on the AGV at the start of the competition. Like I mentioned in the other discussion this trial was more for testing and building out what the configuration file should look like. For assembly you can use the assembly.yaml file. This has been tested with the test competition and all the correct parts are on the AGVs.
